roles/validations/tasks/edpm/hotfix.yml

```diff
@@ -6,86 +6,96 @@
 
 # Since jobs use different names for their OpenStackDataPlaneNodeSet's, we need to dynamically
 # determine the name of the currently deployed one
-- name: Determine name of deployed NodeSet
-  environment:
-    KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
-    PATH: "{{ cifmw_path }}"
-  cifmw.general.ci_script:
-    output_dir: "{{ cifmw_validations_basedir }}/artifacts"
-    script: >-
-      oc get osdpns -n {{ cifmw_validations_namespace }} --no-headers -o custom-columns=":metadata.name"
-  register: deployed_nodeset_name
+- name: Set validator status to passed
+  ansible.builtin.set_fact:
+    validator_status: "passed"
+
+- name: Validation block
+  block:
```

Comment:
Now that we have a block why should we repeat 
```yaml
      environment:
        KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
        PATH: "{{ cifmw_path }}"
```
over and over?

Reply:
Moved the the three environment lines so they applied to the whole block and deleted the rest.